### PR TITLE
[14.0][FIX] account_statement_import_txt_xlsx: handle empty debit/credit

### PR DESCRIPTION
**Pull request by ghost**
When the bank statement import file has two amount columns (in and out), each line is using one column (in or out), and the other column is empty. With this fix, empty values will be handled correctly.

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by norlinhenrik** [comment]
I have rebased now. Can we merge?

**Comment by norlinhenrik** [comment]
Hi @alexey-pelykh
Would you like to review this PR?

**Comment by norlinhenrik** [comment]
@pedrobaeza @MiquelRForgeFlow @alexey-pelykh 
Can we rebase and merge this PR? I am using it in production.

File: account_statement_import_txt_xlsx/tests/fixtures/debit_credit_amount.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 "Date","Label","Debit","Credit","Balance","Partner Name","Bank Account"
-"12/15/2018","Credit 20.00","0.00","20.00","-10.00","John Doe","123456789"
+"12/15/2018","Debit empty, Credit 20",,"20","-10.00","John Doe","123456789"
```
Comment (MiquelRForgeFlow) [inline account_statement_import_txt_xlsx/tests/fixtures/debit_credit_amount.csv]:
```suggestion
"12/15/2018","Debit empty, Credit 20.00",,"20.00","-10.00","John Doe","123456789"
```
🤔

Comment (norlinhenrik) [reply account_statement_import_txt_xlsx/tests/fixtures/debit_credit_amount.csv]: I set 20 to test that it works with an integer. Should I write `Debit empty, Credit integer`?

Comment (ghost) [reply account_statement_import_txt_xlsx/tests/fixtures/debit_credit_amount.csv]: I have now written `Debit test empty, Credit test integer` to be clear on why I have empty / integer values.

**Comment by norlinhenrik** [comment]
Can we merge this now?

**Review by alexey-pelykh — APPROVED** [review]

**Comment by norlinhenrik** [comment]
Can anyone explain the failed check? I don't understand what is the problem.

**Pull request closed without merging**